### DISCUSSION TITLE
How to view the desing-system?

### DISCUSSION THREAD

**Discussion by vitalijalbu** [Thinkmill/keystatic#132] [Q&A] [answered]
Hi guys, was very curious to see the storybook design system locally, how I can run it? Thanks

**Comment by JPrisk** [comment, marked as answer]
Hey @vitalijalbu,

The design system powering Keystatic is still in early development and although much of it is available publicly we don't have the intention of exposing it for general consumption at this time. Stay tuned for future updates!
